traffic_monitor/static/script.js

```diff
@@ -109,7 +109,14 @@ function getTrafficOpsUri() {
 
 function getTrafficOpsCdn() {
 	ajax("/publish/ConfigDoc", function(r) {
-		document.getElementById("cdn-name").textContent = JSON.parse(r).cdnName || "unknown";
+		let opsConfig = JSON.parse(r);
+		document.getElementById("cdn-name").textContent = opsConfig.cdnName || "unknown";
+		var discIconContainer = document.getElementById("icon-disc-holder");
```

Comment:
could be `const`

but also could be calculated once outside of the function instead of on every call.